packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -439,6 +445,24 @@ workflowEventBus.on('node-state-change', (payload: any) => {
 	workflowDirty = true;
 });
 
+workflowEventBus.on('node-refresh', (payload: { workflowId: string; nodeId: string }) => {
+	if (wf.value.id !== payload.workflowId) return;
+	const node = wf.value.nodes.find((n) => n.id === payload.nodeId);
+	if (!node) return;
+
+	if (node.operationType === WorkflowOperationTypes.MODEL) {
+		// This part is a bit hacky and slow. Because we allow multiple instances of the
+		// same model across many nodes in a workflow, they ALL need to be updated. However
+		// this multi-models setup is also somewaht uncommon so I don't want to go out of the way
+		// to communicate "model change" instead of "node change", the former seemingly out of
+		// place when using the WorkflowEventBus mechanism. DC - Aug 2023
+		const nodesToRefresh = wf.value.nodes.filter((n) => n.state.modelId === node.state.modelId);
+		nodesToRefresh.forEach((n2) => {
+			refreshModelNode(n2);
+		});
```

Comment:
If you want to avoid weird `n2` like variable name, just apply your function call to the method.
```suggestion
		nodesToRefresh.forEach(refreshModelNode);
```